frontend/lib/resolution/prims.cpp

```diff
@@ -429,7 +429,7 @@ struct TestFunctionFinder {
     // The function also needs to be concrete.
     const UntypedFnSignature* uSig = UntypedFnSignature::get(context, fn);
     const TypedFnSignature* sig = typedSignatureInitial(context, uSig);
-    if (sig->needsInstantiation()) return false;
+    if (sig != nullptr && sig->needsInstantiation()) return false;
 
     if (canPass(context, testType, sig->formalType(0)).passes()) {
```

Comment:
Regarding checking for nullptrs, if the `sig` above is a nullptr, I'd expect the `seg->formalType(0)` to segfault. Does this need a check too?

Reply:
I suspect this needs to be `||`, not `&&`.